packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1638,5 +1638,13 @@ idx.error.code.access_denied.device_assurance.remediation.chrome.chromeos.upgrad
 idx.error.code.access_denied.device_assurance.remediation.chrome.chromeos.key_trust_level.verified_mode = Switch your device to verified mode
 idx.error.code.access_denied.device_assurance.remediation.chrome.chromeos.key_trust_level.developer_mode = Switch your device to developer mode
 
+# Device Assurance Grace Period
+idx.device_assurance.grace_period.title = Device assurance reminder
+idx.device_assurance.grace_period.explanation_one_rule = Your device does not meet the security requirements. To resolve now, make the following updates. Otherwise, continue to your app.
+idx.device_assurance.grace_period.explanation_multiple_rules = Your device does not meet the security requirements. To resolve now, pick an option and make the updates. Otherwise, continue to your app.
+idx.device_assurance.grace_period.continue_to_app = Continue to app
+idx.device_assurance.grace_period.due_by_days_warning = To prevent account lockout, resolve the issues within {0} days
+idx.device_assurance.grace_period.due_by_date_warning = To prevent account lockout, resolve the issues by {0}
```

Comment:
Is it possible that there is only a issue to resolve? If so, this copy would be slightly awkward since it assumes multiple.

Reply:
Good point, let me check with design

Reply:
A possible workaround for the singular/plural mismatch:

one_rule = Your device doesn't meet the security requirements. You can resolve the noncompliance now. Otherwise, continue to your app.

multiple_rules = Your device doesn't meet the security requirements. To resolve the noncompliance now, pick a remediation option. Otherwise, continue to your app.

To prevent account lockout, resolve the noncompliance within {0} days